CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contribution guide
+
+All contributions are welcome! :)
+
+If you would like to contribute anything, fork the repo and open a Pull Request (PR).
+
+## Recommended git workflow
+
+The best way to ensure that git history is not jumbled up too much is to add an `upstream` remote:
+
+```
+$ git clone ...your_fastats_fork_url...
+$ cd fastats
+$ git remote add upstream https://github.com/fastats/fastats
+```
+
+Then you can fetch from `upstream` remote and create new features on your fork easily:
+
+```
+$ git fetch upstream
+$ git checkout -b my_awesome_branch upstream/master
+```
+
+
+## Issues
+
+Issues are turned off forever.  We prefer Pull Requests for everything.
+
+There's many reasons for this, [this gist][bad_issues] from Ryan Florence details them nicely.
+
+
+## Reporting bugs and requesting changes
+
+- To report a bug, open a PR with a unittest that fails.
+- To request an API change/new functionality, open a PR with a failing unittest showing your
+  preferred API.
+- To submit a fix, open a PR with passing unittests + doctests.
```

Comment:
Maybe worth mentioning that these guidelines themselves are perfectly legitimate to create a PR for